jupyter/datascience/ubi9-python-3.11/Pipfile

```diff
@@ -7,44 +7,50 @@ name = "pypi"
 
 [packages]
 # Datascience and useful extensions
-boto3 = "~=1.34.50"
+boto3 = "~=1.35.5"
 kafka-python = "~=2.0.2"
-matplotlib = "~=3.8.3"
-numpy = "~=1.26.4"
-pandas = "~=2.2.0"
-plotly = "~=5.20.0"
-scikit-learn = "~=1.4.0"
-scipy = "~=1.12.0"
-skl2onnx = "~=1.16.0"
-onnxconverter-common = "~=1.13.0" # Required for skl2onnx, as upgraded version is not compatible with protobuf
+matplotlib = "~=3.9.2"
+numpy = "~=2.1.0"
+pandas = "~=2.2.2"
+plotly = "~=5.23.0"
+scikit-learn = "~=1.5.1"
+scipy = "~=1.14.1"
+skl2onnx = "~=1.17.0"
+onnxconverter-common = "~=1.14.0" # Required for skl2onnx, as upgraded version is not compatible with protobuf
 codeflare-sdk = "~=0.18.0"
 memray = "~=1.10.0"  # any version above this would cause issue with rich package
```

Comment:
Can we upgrade this as well,
and as the memray was included due to codeflare-sdk version, 
can you try and see if removing it gives any issue. 
i believe we can remove it from direct list.

Reply:
Hey I removed memray package, you may inspect it here https://github.com/opendatahub-io/notebooks/pull/687/commits/e0769d2439a4c4b23c780cbe35db088f13043e7f